pkg/eks/eks_test.go

```diff
@@ -290,6 +288,23 @@ var _ = Describe("EKS API wrapper", func() {
 				Expect(p.MockEKS().AssertNumberOfCalls(GinkgoT(), "ListClusters", 1)).To(BeTrue())
 			})
 		})
+
+		Context("`listAllRegions` is true", func() {
+			BeforeEach(func() {
+				chunkSize = 100
+				listAllRegions = true
+
+				p.MockEC2().On("DescribeRegions", mock.Anything).Return(&ec2.DescribeRegionsOutput{}, nil)
+			})
+
+			JustBeforeEach(func() {
+				clusters, err = c.ListClusters(chunkSize, listAllRegions)
+			})
+
+			It("should have called AWS EC2 service once", func() {
+				Expect(p.MockEC2().AssertNumberOfCalls(GinkgoT(), "DescribeRegions", 1)).To(BeTrue())
```

Comment:
I wonder if you could simplify this, given that it will fail the assertion if the number of calls is not 1?

```go
p.MockEC2().AssertNumberOfCalls(GinkgoT(), "DescribeRegions", 1)
```

Reply:
👍 totally right, i think this was another copy-pasta fail from a similar test above 🤦‍♀️